src/frontend/src/pages/CameraDetailsPage.js

```diff
@@ -296,7 +316,7 @@ export default function CameraDetailsPage() {
       </div>
       <div>
         {camera && (
-          <Container className="container--full-width">
+          <container className="container--full-width">
             <div className="camera-details">
```

Comment:
This needs to be reverted - Container needs to be capitalized

Reply:
I went ahead a fixed this since it was probably a linting mistake or typo, not a huge deal.